<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_roi</class>
 <widget class="QWidget" name="tab_roi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QListView" name="roi_lv"/>
     </item>
     <item>
      <widget class="QPushButton" name="register_btn">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
